test/integration/actions.test.js

```diff
@@ -86,7 +86,7 @@ test('create, get and delete an action', t => {
   }).then(result => {
     t.is(result.name, 'random_action_test')
     t.is(result.namespace, NAMESPACE)
-    t.is(result.exec.kind, 'nodejs:6')
+    t.is(result.exec.kind, 'nodejs:10')
```

Comment:
May be we just check if `kind` contains or starts with `nodejs`. Would make test more stable against future changes related to version

Reply:
That's a good suggestion. Let me fix that...